hack/verify.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+ret=0
+bad_files=$(gofmt -s -l awx/ examples/)
+if [[ -n "${bad_files}" ]]; then
+    echo "gofmt needs to be run on the listed files"
+    echo "${bad_files}"
+    echo "Try running 'gofmt -w -d [path]'"
+    ret=1
+fi
+bad_files=$(goimports -l awx/ examples/)
+if [[ -n "${bad_files}" ]]; then
+    echo "goimports needs to be run on the listed files"
+    echo "${bad_files}"
+    echo "Try running 'goimports -w -d [path]'"
+    ret=1
+fi
+
+# Vetting the entirety of the awx package in one go
+go vet ./awx
```

Comment:
vet output will go to STDOUT, please echo something that will tell the user this is go vet output.
Could be:
echo "Running go vet ./awx"
...
echo "Running go vet \"$f'\""